.github/workflows/package.yml

```
@@ -0,0 +1,25 @@
+name: Create release package
+
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  build:
+
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Set env
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+      - uses: actions/checkout@v3
+      - name: Composer install (for indexing of vendor blade classes)
+        run: composer install -q --no-ansi --no-interaction --no-scripts --no-progress --prefer-dist --no-dev
+      - name: Create tar
+        run: tar -czf inge3_${{ env.RELEASE_VERSION }}.tar.gz ./zamex.php ./src ./vendor
```

Templates folder is also needed.

```suggestion
        run: tar -czf inge3_${{ env.RELEASE_VERSION }}.tar.gz ./zamex.php ./src ./templates ./vendor
```